tests/0022-consume_batch.c

```diff
@@ -196,8 +196,68 @@ static void do_test_consume_batch_oauthbearer_cb(void) {
 #endif
 
 
+/**
+ * @brief Subscribe to a non-existent topic with rd_kafka_consume_batch_queue.
+ *        Verify that a rkmessage with error code ERR_UNKNOWN_TOPIC_OR_PART
+ *        is received.
+ */
+static void do_test_consume_batch_non_existent_topic(void) {
+
+        char *topic = "non-existent2";
```

Comment:
Just curious - Why 2 in "non-existent2"?

Reply:
Probably stale name :smile: , changed it